src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Parse class that handles the string inputted by user.
+ */
+public class Parser {
+    private Duke duke;
+
+    /**
+     * Constructor for Parser Class.
+     *
+     * @param duke The duke object the user is operating on.
+     */
+    public Parser(Duke duke) {
+        this.duke = duke;
+    }
+
+    /**
+     * Parses the various inputs inputted by user.
+     */
+    public void start() {
```

Comment:
Perhaps it might be clearer if the method name was more descriptive? eg. `parseUserInputs()`